Src/IronPython/Runtime/Converter.cs

```diff
@@ -336,23 +336,16 @@ public static partial class Converter {
             return res.HasValue;
         }
 
-        internal static bool TryConvertToIndex(object value, out object index) {
-            return TryConvertToIndex(value, true, out index);
-        }
-
         /// <summary>
-        /// Attempts to convert value into a index usable for slicing and return the integer
+        /// Attempts to convert value into an index usable for slicing and return the integer or BigInteger
         /// value.  If the conversion fails false is returned.
-        /// 
-        /// If throwOverflowError is true then BigInteger's outside the normal range of integers will
-        /// result in an OverflowError.
         /// </summary>
-        internal static bool TryConvertToIndex(object value, bool throwOverflowError, out object index) {
-            index = ConvertToSliceIndexHelper(value, throwOverflowError);
+        internal static bool TryConvertToIndex(object value, out object index) {
```

Comment:
Just did a quick search and it looks like this is called in a bunch of places. Does this mean that they will no longer throw?

Reply:
As far as I can see, this overload is only called in `Enumerate.cs`. The other overload is called in many places but I haven't touched that one.

Reply:
Ah I see, the others use the int overload.